factory/src/contract.rs

```diff
@@ -1,192 +1,159 @@
 use dex_factory_io::*;
-use dex_pair_io::InitPair;
-use gstd::{errors::Result, exec, msg, prelude::*, prog::ProgramGenerator, ActorId, MessageId};
-use hashbrown::HashMap;
-
-#[derive(Default)]
-struct Factory {
-    pair_code_hash: [u8; 32],
-    owner_id: ActorId,
+use gstd::{
+    errors::Result as GstdResult, exec, msg, prelude::*, prog::ProgramGenerator, ActorId, CodeId,
```

Comment:
I guess we don't need to differentiate gstd::Result after https://github.com/gear-tech/gear/pull/2333
```suggestion
    errors::Result, exec, msg, prelude::*, prog::ProgramGenerator, ActorId, CodeId,
```